### ISSUE TITLE
installation error in ubuntu 18.04

### ISSUE THREAD

**Issue by rpotter12** [apache/couchdb#2377]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

I was unable to install CouchDB on my system. I was following steps of http://docs.couchdb.org/en/latest/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages

I have used following commands to install:
```
$ sudo apt-get install -y apt-transport-https gnupg ca-certificates
$ echo "deb https://apache.bintray.com/couchdb-deb bionic main" \
    | sudo tee -a /etc/apt/sources.list.d/couchdb.list
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys \
  8756C4F765C9AC3CB6B85D62379CE192D401AB61
$ sudo apt update
$ sudo apt install -y couchdb
```

The result for `sudo apt install -y couchdb` was:
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 couchdb : Depends: libicu60 (>= 60.1-1~) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )
[TIP]:  # ( You can paste the output of curl http://YOUR-COUCHDB:5984/ here. )

* CouchDB Version used:
* Browser name and version:
* Operating System and version: Ubuntu 18.04

**Comment by maojianyu** [comment]
It show the error `couchdb : Depends: libicu60 (>= 60.1-1~) but it is not installable`. So you should install the `libicu` and `libicu-dev` library.
```
$ sudo apt-get intall libicu
$ sudo apt-get intall libicu-dev
```

**Comment by rpotter12** [comment]
@jyxiaoka 
results for `sudo apt install ilbicu`: 
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libicu
```

and for `sudo apt install libicu60`:
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libicu60 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libicu60' has no installation candidate
```

how to solve this?

**Comment by wohali** [comment]
https://packages.ubuntu.com/bionic/libicu60 says the package is available for 18.04.

Check your `/etc/apt/sources.list` file and ensure it has the correct line in it, such as:

```
deb http://us.archive.ubuntu.com/ubuntu/ bionic main restricted
```

Then do `sudo apt update && sudo apt install libicu60`.
